test/Conductors.Tests/RepositoryConductorTests/DeleteAsyncShould.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using AndcultureCode.CSharp.Core.Interfaces.Conductors;
+using AndcultureCode.CSharp.Core.Models.Errors;
+using AndcultureCode.CSharp.Testing.Models.Stubs;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace AndcultureCode.CSharp.Conductors.Tests.RepositoryConductorTests
+{
+    public class DeleteAsyncShould : ProjectUnitTest
+    {
+        public DeleteAsyncShould(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        private IRepositoryConductor<UserStub> SetupSut(
+            RepositoryMock<UserStub> repositoryMock
+        )
+        {
+            var repositoryDeleteConductor = new RepositoryDeleteConductor<UserStub>(repositoryMock.Object);
+            return new RepositoryConductor<UserStub>(
+                createConductor: null,
+                deleteConductor: repositoryDeleteConductor,
+                readConductor: null,
+                updateConductor: null
+            );
+        }
+
+        [Fact]
+        public async Task Succeed_When_Given_Proper_Id()
+        {
+            // Arrange 
+            var repositoryMock = new RepositoryMock<UserStub>()
+                .DeleteAsync(new Result<bool>(true));
+            var respositoryConductor = SetupSut(repositoryMock);
+
+
```

Comment:
Minor: The spacing before and after Arrange/Act/Assert comments is a tad inconsistent in this file. My pattern-matching brain is breaking ;)

Reply:
your pattern matching brain should now be soothed :D